### PR TITLE
[Refactor] Add Webcam API unittests

### PR DESCRIPTION
**Pull request by Ben-Louis**
## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Comment by ly015** [comment]
#1638 has been merged. Please rebase the pr branch onto the latest dev-1.x

**Comment by Ben-Louis** [comment]
The unittest for the missed lines, indeed the unittest for `webcam_executor.run`, involves multi-threading. 
It will cause `segmentation fault` in `pr_stage_test / build_cpu (3.7)` and fail while pass in other environments. Is there any method to fix this CI problem? @ly015

File: mmpose/apis/webcam/utils/pose.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 
 def get_eye_keypoint_ids(dataset_meta: Dict) -> Tuple[int, int]:
-    """A helpfer function to get the keypoint indices of left and right eyes
-    from the dataset meta information.
+    """A help function to get the keypoint indices of left and right eyes from
```
Comment (ly015) [inline mmpose/apis/webcam/utils/pose.py:6]: I think the term 'helper function' is ok. Is there concern about it?

Comment (Ben-Louis) [reply mmpose/apis/webcam/utils/pose.py:6]: Actually not. I found the typo 'helpfer' and simply replaced it with 'help'. I'll change it now. 

BTW, maybe we also need to fix this type in master?

**Review by ly015 — APPROVED** [review]

**Pull request merged**